### PR TITLE
9 viewing your current listings

### LINKED ISSUE #9: viewing your current listings
```
As a user
So I can keep track of my listings
I'd like to be able to see all my listings.
```

### PR DESCRIPTION
**Pull request by reissjohnson**
closes #9

### CODE REVIEW

**Comment by frazerWatson** [comment]
:shipit:

**Comment by sjmog** [comment]
Just to say - this process is so good, it brings tears to my eyes

- Squirrels
- correctly named branch
- correct closure for integration with Waffle
- correct labelling (QA)
- clear commit history
- User Story in the ticket

I’m so freaking happy right now

File: lib/my_app.rb [hunk 1]
```diff
@@ -23,8 +23,10 @@ def current_user
   end
 
   post '/' do
-  	Space.create(location: params[:location],description: params[:description], price: params[:price], name: params[:name], details: params[:details] )    
-        redirect to('/new')
+  	a_space = Space.create(location: params[:location],description: params[:description], price: params[:price], name: params[:name], details: params[:details] )
```
Comment (sjmog) [inline lib/my_app.rb]: What happens if you just pass `params`? You're constructing an identical hash :dango:

File: spec/features/viewing_list_of_users_spaces_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+end
+  	# post '/links' do
+  	# 	link = Link.new(url: params[:url], title: params[:title])
+   #    params[:tags].split.each do |tag|
+   #    tag = Tag.create(name: tag)
+   #    link.tags << tag 
+   #  end
+   #    link.save
+  	# 	redirect('/links')
+   #  endwithin 'ul#links' do
```
Comment (sjmog) [inline spec/features/viewing_list_of_users_spaces_spec.rb]: Comments in commits :scream:

File: lib/my_app.rb (hunk 1 above)
Comment (katie210) [inline lib/my_app.rb]: Oh, wow. I see now! Didn't think DataMapper would handle it that way. Thanks for the suggestion!

Comment (sjmog) [inline lib/my_app.rb]: It ain't DataMapper

```ruby
params == { location: something, blah: something_else }
```

What you're constructing in that line:

```ruby
arguments_to_space_create == { location: something, blah: something_else }
```

i.e. the params hash is the same as the arguments hash you're passing to `create`.

**Comment by katie210** [comment]
:shipit:

**Pull request merged**